docs/development/l10n.rst

```diff
@@ -0,0 +1,93 @@
+Getting Started with Weblate translations
+=========================================
+
+SecureDrop is a whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. For more information, you may be interested to learn about :doc:`what makes SecureDrop unique <../what_makes_securedrop_unique>`.
+
+SecureDrop uses the Weblate platform. This page lists detailed instructions for getting started with the Weblate platform as a translator. Before starting this guide, you should know which LANGUAGE you would like to translate.
+
+.. _`Weblate registration page`: https://weblate.securedrop.club/accounts/register/
+
```

Comment:
Updated with 

Who uses SecureDrop ?
---------------------

There are two kind of SecureDrop users: :doc:`Sources <../source>` and :doc:`Journalists <../journalist>`. A source is an individual who wants to communicate securily and anonymously with a journalist. Sources are not expected to be have any technical background. Journalists using SecureDrop have all received proper training and know what a PGP public key is, they understand the basic workflow of SecureDrop.